xds/internal/client/lds.go

```diff
@@ -85,6 +86,7 @@ func getRouteConfigNameFromListener(lis *xdspb.Listener) (string, error) {
 	if !ok {
 		return "", fmt.Errorf("xds: unexpected api_listener type: %T in LDS response", apiAny.Message)
 	}
+	infof(parent, "Resource with type %T, contains %v", apiLis, apiLis)
```

Comment:
Could we just dump the `listener proto` before the call to `getRouteConfigNameFromListener`, thereby avoiding passing the `parent` pointer to this function?